src/components/platformSidebar.tsx

```diff
@@ -90,6 +91,15 @@ export const PlatformSidebar = ({
         suppressMissing
         tree={tree}
       />
+      <DynamicNav
+        root={`/${pathRoot}/session-replay`}
+        title="Session Replay"
+        prependLinks={[
+          [`/${pathRoot}/session-replay/`, "Set Up Session Replay"],
+        ]}
+        suppressMissing
+        tree={tree}
+      />
```

Comment:
SDK nav: 
(the exclude list above means that the link will not appear (again) under the "Sentry for JavaScript" heading)

<img width="290" alt="SCR-20221205-nsp" src="https://user-images.githubusercontent.com/187460/205781017-4c51e173-9186-49b9-951f-be5a589366be.png">